frontend/src/components/pages/SearchReviews/ReviewThumbnail.tsx

```diff
@@ -66,9 +68,17 @@ const ReviewThumbnail = (props: ReviewThumbnailProps): React.ReactElement => {
       >
         {authors.map((elem) => elem.displayName).join(", ")}
       </Text>
-      <Text textStyle={["mobileCaption", "mobileCaption", "caption"]} mt="2%">
-        {publishDate.toLocaleDateString("sv")}
-      </Text>
+      <Flex>
+        <Text textStyle={["mobileCaption", "mobileCaption", "caption"]} mt="2%">
+          {publishDate.toLocaleDateString("sv")}
+        </Text>
+        <Spacer />
+        {genre && (
+          <Tag size="sm" variant="solid" bgColor="#F7E1A8" color="black">
```

Comment:
Just realized I send the outdated Figma design mb. This is actually supposed to be blue (ie the same as the genre colour in the review display)

Reply:
Fixed